<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMainWidget</class>
 <widget class="QWidget" name="frmMainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>frmMainWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="btnDispense">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="text">
      <string>配药</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnRecipeSet">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="text">
      <string>配药设置</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnSystemSet">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="text">
      <string>系统设置</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnExit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
